modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -51,6 +46,8 @@ const RichText = ({
 		}
 	}, [editingLanguageId, editorRef]);
 
+	const currentValue = predefinedValue || value;
```

Comment:
- I believe `value` should came first, so the `predefinedValue` will be used when no value is provide. 
- Another point to attention is the use or the operator OR (`||`),  use the [Nullish coalescingoperator (??)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator), so once the input value is changed `value` will be display in place of the `predefinedValue` (even when it is an empty string).

```suggestion
	const currentValue = value ?? predefinedValue;
```

Reply:
Hi, @miltonmc I will do the changes!